### PR TITLE
Add option to control verbosity level with command line args.

### PR DESCRIPTION
**Pull request by pranaysashank**
- Adds `-v0`, `-v1`, `-v2`, `-verbose=warn` to control verbosity level
- Changes `dump-core` to `-ddump-core`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## Unreleased
+
+### Enhancements
```
Comment (pranaysashank) [inline CHANGELOG.md:3]: @harendra-kumar Can you comment / add the Changelog entry for the new plugin pass

**Comment by harendra-kumar** [comment]
If we rename the `dump-core` option we should keep the original one as well with a deprecation message. It does not make much difference we can just keep the old one.

I removed the shorter versions, as of now we can just keep it to minimum, anyway adding them later is still open and can be done easily at any time.

**Review by harendra-kumar — APPROVED** [review]
I pushed proposed changes. Please review.

**Comment by pranaysashank** [comment]
@harendra-kumar In function `markInline`, `ReportWarn` reports warnings that don't get reported in other verbosity levels, when I pass verbose=2 or 3 I expect not to lose info that is contained in level 0. With your change this doesn't happen

**Comment by harendra-kumar** [comment]
I guess, we can keep that information in higher verbosity levels. We can discuss/track that in a separate issue.

File: CHANGELOG.md (hunk 1 above)
Comment (harendra-kumar) [reply CHANGELOG.md:3]: - Now forces inline on binders that return a type annotated with `Fuse`.

**Pull request merged**